apps/masterbots.ai/app/actions/ai-executers.ts

```
@@ -0,0 +1,275 @@
+      throw new Error(data.error || 'Failed to fetch prompt details')
+    }
+    inputs = data.inputs.reduce(
+      (acc: any, input: { label: any }) => ({
+        ...acc,
+        [input.label]: ''
+      }),
+      {}
+    )
+  } catch (error) {
+    console.error('Error fetching prompt details:', error)
+    error = (error as Error).message
```

Biome

[error] Reassigning a catch parameter is confusing.

The catch parameter is declared here:

Use a local variable instead.

(lint/suspicious/noCatchAssign)